### ISSUE TITLE
Schnelleinstieg für einfachstes Setup, nämlich GUI in der Cloud

### ISSUE THREAD

**Issue by jokaiserMPIC** [CCI-Tools/cate#1015]
When I try tu use CATE for the first time, I am going for the least invasive setup (from my perspective). This is using the GUI in the cloud from a web browser. In order to do this I am trying to follow the documentation suggested at https://climate.esa.int/de/explore/analyse-climate-data/. Here the easiest documentation is presumably behind the link "Schnelleinstieg". I expect to find brief introductory instruction how to use CATE with the mentioned setup. However, only the CLI and API are described, not the GUI.
May I suggest that you add instructions that allow a novice to CATE to achieve results with the GUI and CATE running in the cloud.

**Comment by forman** [comment]
> May I suggest that you add instructions that allow a novice to CATE to achieve results with the GUI and CATE running in the cloud.

Yes, we are aware that this would be very useful. We provided a tutorial for the desktop version, but it hasn't yet been updated for the Cate cloud service. Find that one here: https://youtu.be/w0ZEK4aIC9I
